project2.qmd

```diff
@@ -111,6 +111,119 @@ print(combined_plot)
 cat('<a href="https://afraser.shinyapps.io/shiny/" target="_blank" onclick="window.open(\'https://afraser.shinyapps.io/shiny/\', \'newwindow\', \'width=600,height=600\'); return false;">Launch the popup</a> to review individual variable boxplots.')
 ```
 
+## Multicollinearity
+
+### VIF & linear regression
+
+```{r message=FALSE, warning=FALSE}
+library(car)
+
+lm_vif <- lm(ph ~ ., data = train)
+vif(lm_vif)
+
+summary(lm_vif)
+```
+
+```{r message=FALSE, warning=FALSE}
+library(PerformanceAnalytics)
+
+# List of selected variables based on GVIF criteria
+selected_vars <- c("carbpressure", "carbtemp", "fillerlevel", 
```

Comment:
> table(train$carbpressure)
< table of extent 0 >
Warning message:
Unknown or uninitialised column: `carbpressure`. 


This isn't a variable, right? There's a carbpressure1.